src/features/home/CardOfDish.tsx

```diff
@@ -10,15 +11,15 @@ interface Props {
 }
 
 export const CardOfDish: React.FC<Props> = ({dish, onPress}: Props) => (
-  <TouchableOpacity style={styles.dishesBlock} onPress={() => onPress(dish)}>
-    <View style={styles.dishesItem}>
+  <View style={styles.dishesBlock}>
```

Comment:
@DiamondRock597 Again useless View wrapper around ToucahbleOpacity, we can use style from ToucahbleOpacity. (just merge styles and check if its correct if not try to fix)